src/leafdoc.js

```diff
@@ -567,7 +576,17 @@ Leafdoc.prototype._stringifyNamespace = function(namespace, isMini) {
 	});
 };
 
-
+/*
+ * 🍂method isDocumentableInheritable(name: String): Boolean
+ * Determines documentable can be inherited.
+ */
+Leafdoc.prototype.isDocumentableInheritable = function(name) {
```

Comment:
For myself I use following code:

```es6
class ApiLeafdoc extends LeafDoc {
  isDocumentableInheritable(name) {
    return super.isDocumentableInheritable(name) || name === 'field';
  }
}
```

I don't mind having inheritable documentables in array, in this case we can simplify code for add new inheritable documentables:

```js
doc.inheritableDocumentables.push('item');
```

What do you think ? Shall I use this option?